android/src/main/java/com/th3rdwave/safeareacontext/SafeAreaUtils.kt

```diff
@@ -72,10 +71,18 @@ fun getSafeAreaInsets(view: View): EdgeInsets? {
   val visibleRect = android.graphics.Rect()
   view.getGlobalVisibleRect(visibleRect)
   return EdgeInsets(
-      top = max(windowInsets.top - visibleRect.top, 0f),
-      right = max(min(visibleRect.left + view.width - windowWidth, 0f) + windowInsets.right, 0f),
-      bottom = max(min(visibleRect.top + view.height - windowHeight, 0f) + windowInsets.bottom, 0f),
-      left = max(windowInsets.left - visibleRect.left, 0f))
+      top = maxOf(windowInsets.top - visibleRect.top, windowInsets.top, 0f),
+      right =
+          maxOf(
+              min(visibleRect.left + view.width - windowWidth, 0f) + windowInsets.right,
```

Comment:
If we need to use `maxOf`, could we also use `minOf` for consistency?

Reply:
Sure!

Reply:
Resolved in https://github.com/th3rdwave/react-native-safe-area-context/pull/422/commits/cbb667ccc9d1b69b9416a1310b23fb5dee16ff3c.